src/components/NavBar/NavBarLogo.tsx

```diff
@@ -1,21 +1,16 @@
 import { Link } from 'react-router-dom';
 import OfekLogo from '../../assets/images/OfekLogo.png';
+import LogoStyle from './Logo.module.css';
 
 interface LogoPropsType {
   header: string;
 }
 const NavBarLogo = ({ header }: LogoPropsType) => (
```

Comment:
you can't logically explain  you navBarlogo has link to root path, rename to navLogoLink maybe ( I'm  not 100% sure on this take so if you disagree pop emoji is suitable )